plugins/kubernetes/app/controllers/kubernetes/clusters_controller.rb

```diff
@@ -121,6 +84,6 @@ def secret_exist?(secret)
   end
 
   def secrets_client
-    @cluster.client('v1')
+    @kubernetes_cluster.client('v1')
```

Comment:
In the future should the client version be a parameter or env var?

Reply:
secrets are v1 .. so either it's a constant or just stays this way 🤷‍♂️